packages/services/f-feature-management/.eslintrc.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  extends: [
+    // add more generic rulesets here, such as:
+    'plugin:json-schema-validator/base'
+  ],
+  overrides: [
+    {
+      files: ["*.json", "*.jsonc", "*.json5", "*.json6"],
+      parserOptions: {
+        parser: 'jsonc-eslint-parser',
+      }
+    }
+  ],
+  rules: {
+    // json-schema-validator plugin does not play nicely with the vuejs-accessibility plugin.
```

Comment:
As discussed @ashleynolan

Reply:
This has now been fixed for `eslint-plugin-vuejs-accessibility@v0.7.1` - will update in next PR and hopefully remove all these exceptions 🙌 

FYI @xander-marjoram